packages/app/src/server/form/index.js

```diff
@@ -1,6 +1,7 @@
 module.exports = {
   login: require('./login'),
   register: require('./register'),
+  registerUserActivation: require('./registerUserActivation'),
```

Comment:
Don't use express-form because deprecated in GROWI system.
Use express-validator instead.